credible/credible-layer-overview.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: 'Credible Layer Overview'
+description: 'This guide explains each aspect of the Credible Layer system to give you an accurate picture of how exploit prevention works, without requiring deep technical knowledge.'
+---
+
+## Introduction
+
+Phylax's real-time security system detects and pre-emptively removes exploits before losses can be realized.
+
+![System Overview](/images/overview-step-0.png)
+
+At a glance:
+
+- **[Invariants](/credible/glossary#protocol-invariants)** are mapped and defined by the DeFi protocol/dapp team
+- **Invariants** are converted into **[assertions](/credible/glossary#assertion)** so that the network node knows when to [trigger](/credible/glossary#trigger) and perform validation
```

Comment:
The team converts their invariants (or other rules such as KYC/AML, etc) into assertions, which are used by the network node to validate if a transaction is valid or invalid.

Also, you can't use network node here but later use the node sidecar, since it's the same component that does the enforcement. You have to choose whether you want to be specific (sidecar) or network node in a more abstract way'